src/modules/task/task.controller.ts

```diff
@@ -18,20 +18,20 @@ export class TaskController {
     this.router.delete('/:id', (req, res) => this.deleteTasksId(req, res));
   }
 
-  create(req: Request, res: Response) {
+  async create(req: Request, res: Response) {
     const dto = validate(CreateTaskDto, req.body);
-    const result = this.service.create(dto);
+    const result = await this.service.create(dto);
     res.json(result);
   }
 
-  getTasks(req: Request, res: Response) {
-    const result = this.service.getTasks();
+  async getTasks(req: Request, res: Response) {
+    const result = await this.service.getTasks();
     res.json(result);
   }
 
-  getTasksId(req: Request, res: Response) {
+  async getTasksId(req: Request, res: Response) {
     const dto = validate(TaskIdDto, req.params);
-    const result = this.service.getTaskId(dto);
+    const result = await this.service.getTaskId(dto);
```

Comment:
Смотри ниже, предложил упростить просто до id:
```suggestion
    const result = await this.service.getTaskId(dto.id);
```